packages/core/src/Stores/client-store.js

```diff
@@ -123,6 +123,7 @@ export default class ClientStore extends BaseStore {
      */
     @computed
     get is_client_allowed_to_visit() {
+        // TODO: [deriv-eu] Remove this after complete EU merge into production
```

Comment:
Redundant comment.

Reply:
OK. What about identical comments in `app-modals` and `purchase`?